src/libraries/System.Text.Json/src/System/Text/Json/Serialization/JsonConverterOfT.cs

```diff
@@ -433,5 +431,14 @@ internal void VerifyWrite(int originalDepth, Utf8JsonWriter writer)
         /// <param name="value">The value to convert.</param>
         /// <param name="options">The <see cref="JsonSerializerOptions"/> being used.</param>
         public abstract void Write(Utf8JsonWriter writer, T value, JsonSerializerOptions options);
+
+        internal virtual T ReadWithQuotes(ref Utf8JsonReader reader)
```

Comment:
I believe we previously discussed this, but if someone creates a custom converter such as for Int32 then they will get the default semantics when using quoted keys, not the custom semantics, correct?

If\when we want to support custom converters for quoted keys, then we can add support for exposing `ReadWithQuotes` publicly or find some other way to do this.